### PR TITLE
[android] Set required NDK in build.gradle (the latest version does not compile yet)

### PR DESCRIPTION
**Pull request by biodranik**
TODO: Integrate the latest Android NDK

### CODE REVIEW

**Review by vng — CHANGES_REQUESTED** [review]
Manual NDK is ok, but why we manually set cmake version?
We can remove cmake:version IMHO.

**Comment by biodranik** [comment]
There are many cool options in newer cmake I want to use.

**Comment by biodranik** [comment]
In this way we'll have predictable environments. Cmake with this version is already bundled in Android SDK.

**Review by vng — APPROVED** [review]

**Comment by vng** [comment]
Here is CI check error:
```
CMake '3.18.1' or higher was not found in PATH or by cmake.dir property.
  - CMake '3.10.2' found in SDK could not satisfy requested version '3.18.1' because it was lower.
  - CMake '3.10.2' found in PATH could not satisfy requested version '3.18.1' because it was lower.
  ```

File: .github/workflows/android-beta.yaml [hunk 1]
```diff
@@ -41,6 +41,7 @@ jobs:
         shell: bash
         run: |
           (cd tools/android; ./set_up_android.py --sdk $ANDROID_SDK_ROOT --ndk $ANDROID_NDK_ROOT)
+          (echo "yes" | $ANDROID_HOME/tools/bin/sdkmanager "cmake;3.18.1")
```
Comment (rtsisyk) [inline .github/workflows/android-beta.yaml:44]: I don't know how to explain this, but it works via Android Studio, but doesn't work via command-line.

**Review by rtsisyk — CHANGES_REQUESTED** [review]
Please manage to fix CI.

**Comment by biodranik** [comment]
OK, removed cmake commits. We'll upgrade cmake later.

**Pull request merged**